include/lsst/meas/extensions/photometryKron/KronFluxAlgorithm.h

```diff
@@ -0,0 +1,130 @@
+                       "if true, use existing shape and centroid measurements instead of fitting");
+    LSST_CONTROL_FIELD(nSigmaForRadius, double,
+                       "Multiplier of rms size for aperture used to initially estimate the Kron radius");
+    LSST_CONTROL_FIELD(nIterForRadius, int, "Number of times to iterate when setting the Kron radius");
+    LSST_CONTROL_FIELD(nRadiusForFlux, double, "Number of Kron radii for Kron flux");
+    LSST_CONTROL_FIELD(maxSincRadius, double,
+                       "Largest aperture for which to use the slow, accurate, sinc aperture code");
+    LSST_CONTROL_FIELD(minimumRadius, double,
+                       "Minimum Kron radius (if == 0.0 use PSF's Kron radius). "
+                       "Ignored if enforceMinimumRadius is false");
+    LSST_CONTROL_FIELD(enforceMinimumRadius, bool, "If true check that the Kron radius exceeds some minimum");
+    LSST_CONTROL_FIELD(useFootprintRadius, bool,
+                       "Use the Footprint size as part of initial estimate of Kron radius");
+    LSST_CONTROL_FIELD(smoothingSigma, double,
+                       "Smooth image with N(0, smoothingSigma^2) Gaussian while estimating R_K");
+    KronFluxControl() :
+        fixed(false),
+        nSigmaForRadius(6.0),
+        nIterForRadius(1),
+        nRadiusForFlux(2.5),
+        maxSincRadius(10.0),
+        minimumRadius(0.0),
+        enforceMinimumRadius(true),
+        useFootprintRadius(false),
+        smoothingSigma(-1.0)
+    {}
+};
+/**
+ *  @brief A measurement algorithm that estimates flux using Kron photometry
+ */
+class KronFluxAlgorithm : public base::SimpleAlgorithm {
+public:
+
+    enum {
+        FAILURE=base::FlagHandler::FAILURE,
+        RADIUS,
+        SMALL_RADIUS,
+        N_FLAGS
+    };
+
+    /// A typedef to the Control object for this algorithm, defined above.
+    /// The control object contains the configuration parameters for this algorithm.
+    typedef KronFluxControl Control;
+
+    KronFluxAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema);
+
+private:
+
+    virtual void measure(
+        afw::table::SourceRecord & measRecord,
+        afw::image::Exposure<float> const & exposure
+    ) const;
+
+    virtual void fail(
+        afw::table::SourceRecord & measRecord,
+        meas::base::MeasurementError * error=NULL
+    ) const;
+
+
+    void _applyAperture(
+        afw::table::SourceRecord & source,
+        afw::image::Exposure<float> const& exposure,
+        KronAperture const& aperture
+        ) const;
+
+    void _applyForced(
+        afw::table::SourceRecord & source,
+        afw::image::Exposure<float> const & exposure,
+        afw::geom::Point2D const & center,
+        afw::table::SourceRecord const & reference,
+        afw::geom::AffineTransform const & refToMeas
+    ) const;
+    std::string const & _name;
+    Control _ctrl;
+    //algorithms::ScaledFlux::KeyTuple _fluxCorrectionKeys;
```

Comment:
Why is this commented?

(I guess the answer is that it relates to the commented "FIXME" lines in `KronFluxAlgorithm.cc`; we'll consider those lines when we get there, but this line definitely shouldn't be left commented with no explanation.)